material-popup-menu/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="copy">Copy</string>
```

Comment:
This is used in the sample app only so it shouldn't be in the library.
Could you move it to `sample/src/main/res/values/strings.xml` instead, please?

Reply:
Done.